mongodb/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mongo:latest
+
+COPY init-mongo.js /docker-entrypoint-initdb.d/init-mongo.js
```

Comment:
Is this possiible to change as ConfigMap of K8s manifests. I thought it's better to use as configmap since we don't need to publish container to Container Registry when we want to change the initial data if that is possible.

Reply:
I understand.